### ISSUE TITLE
SwiftFormat wrongly transforming `fileprivate` into `private`

### ISSUE THREAD

**Issue by vincentisambart** [nicklockwood/SwiftFormat#478]
SwiftFormat 0.40.11 transforms the following Swift 5 code:
```swift
import Foundation

struct MyStruct {
    fileprivate enum Inner {
        case a, b
    }

    fileprivate let inner: Inner
}

func foo(z: MyStruct) {
    print(z.inner)
}
```
to the following code that doesn't compile anymore: (`Inner`'s `fileprivate` became `private`)
```swift
import Foundation

struct MyStruct {
    private enum Inner {
        case a, b
    }

    fileprivate let inner: Inner
}

func foo(z: MyStruct) {
    print(z.inner)
}
```

**Comment by nicklockwood** [comment]
Thanks for reporting. Fixed in 0.40.12.

**Comment by vincentisambart** [comment]
Thanks!
